From: Carlos Garnacho <carlosg@gnome.org>
Date: Tue, 17 Nov 2020 14:55:14 +0100
Subject: clutter/main: Drop usage of clutter_input_device_get_device_id()

Just print the name in those debug logs.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1569>
---
 clutter/clutter/clutter-main.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/clutter/clutter/clutter-main.c b/clutter/clutter/clutter-main.c
index 0a21d1f..459d07d 100644
--- a/clutter/clutter/clutter-main.c
+++ b/clutter/clutter/clutter-main.c
@@ -1319,9 +1319,9 @@ event_click_count_generate (ClutterEvent *event)
       previous_button_number = device->previous_button_number;
 
       CLUTTER_NOTE (EVENT,
-                    "Restoring previous click count:%d (device:%d, time:%u)",
+                    "Restoring previous click count:%d (device:%s, time:%u)",
                     click_count,
-                    clutter_input_device_get_device_id (device),
+                    clutter_input_device_get_device_name (device),
                     previous_time);
     }
   else
@@ -1375,9 +1375,9 @@ event_click_count_generate (ClutterEvent *event)
 
   if (event->type == CLUTTER_BUTTON_PRESS && device != NULL)
     {
-      CLUTTER_NOTE (EVENT, "Storing click count: %d (device:%d, time:%u)",
+      CLUTTER_NOTE (EVENT, "Storing click count: %d (device:%s, time:%u)",
                     click_count,
-                    clutter_input_device_get_device_id (device),
+                    clutter_input_device_get_device_name (device),
                     previous_time);
 
       device->click_count = click_count;
